INSTALL-NOTES-CMAKE

```diff
@@ -0,0 +1,323 @@
+(standard Unix make will NOT work) to build the libraries. 
+
+	(a) cd objs
+	(b) gmake
+
+At any time if you want to start over you may remove all object and library
+files via ``gmake clean''.
+
+(4) Making the tools
+------------------------
+
+The optional tools may be built if you add this to the cmake line:
+
+   -DENABLE_TOOLS=ON 
+
+This includes the restart-redistribute utility to convert a SAMRAI
+restart file to run on N processors to one compatible with running on
+M processors.
+
+
+(6) Installing the Libraries
+----------------------------
+
+After the libraries have been created, install the libraries and configuration
+files using:
+
+	(a) gmake install
+
+For this to work, you must have used the ``-DCMAKE_INSTALL_PREFIX=[directory]''
+to specify where you want your installed files to be located.
+``gmake install'' will move files into the specified directory.
+
+(6) Cleaning Up
+---------------
+
+After installing the libraries, you may remove the source and 
+object directory ``objs'' in which SAMRAI was compiled.  However, 
+it is recommended that you do not remove the SAMRAI source
+directory, since it contains source files for the debugger.
+
+
+CMake Options:
+-------------------
+
+There are many CMake options that can be put on the CMake line.  Here we
+list some of the most useful options.
+
+Compilers:
+
+  -DCMAKE_CXX_COMPILER=[C++ compiler]
+  -DCMAKE_Fortran_COMPILER=[Fortran compiler]
+
+Libraries:
+
+Setting the optional third-party libraries follows a straightforward pattern.
+
+ -DHDF5_DIR=[HDF5 directory location]
+ -DHYPRE_DIR=[HYPRE directory location]
+ -DPETSC_DIR=[PESTC directory location]
+ -DSILO_DIR=[SILO directory location]
+ -DSUNDIALS_DIR=[Sundials directory location]
+ -DCONDUIT_DIR=[Conduit directory location]
+
+HDF5 is the only one of these that is included by default.  CMake usually will
+be able to find an HDF5 installation without one being specified on the
+command line, but this option allows you to point explicitly to an
+installation.
+
+These libraries can also be explicitly excluded using
+
+ -DENABLE_HDF5=OFF
+ -DENABLE_HYPRE=OFF
+ -DENABLE_PETSC=OFF
+ -DENABLE_SILO=OFF
+ -DENABLE_SUNDIALS=OFF
+ -DENABLE_CONDUIT=OFF
+
+Build options:
+
+This CMake option governs usage of optimization or debugging flags:
+
+ -DCMAKE_BUILD_TYPE=[Release|Debug|RelWithDebInfo]
+
+Choose whether to build executables or just the libraries:
+
+ -DENABLE_TESTS=[ON|OFF]  (Default ON)
+ -DENABLE_TOOLS=[ON|OFF]  (Default OFF)
+
+Installation options:
+
+ -DCMAKE_INSTALL_PREFIX=[installation directory]
+
+Documentation:
+
+ To enable the build of documenation using doxygen, add:
+
+ -DENABLE_DOCS=ON
+ 
+
+Versions of libraries/tools used by the development team
+--------------------------------------------------------
+
+The following are the versions of libraries and tools we have used in
+the development of this release.  This list should not be considered
```

Comment:
This notes file references "this release". Is it rewritten for every release then?
```suggestion
the development of the library.  This list should not be considered
```

Reply:
This section should be revised every release to make sure that the third-party library versions listed are up to date.  Still this is a good suggestion to keep the wording more accurate in general.  I'll will change it after this pull request, because I shouldn't change the release branch.